backend/cdb_csv/pe.py

```diff
@@ -414,7 +416,7 @@ async def save_external_data(
 
     # If we have no external_data or we already have some,
     # but the hash is not the same
-    if external_data is None or hash_result != external_data.hash:
+    if external_data is None or hash_result != external_data.hash or not check_hash:
```

Comment:
ca pourrait être plus clair d'inverser la condition et retourner `external_data`

ainsi on gagnerait un niveau d'indentation.

```suggestion
    if external_data is None or hash_result != external_data.hash or not check_hash:
```

Reply:
Désolé pas compris ce que tu veux dire ?

Reply:
Je voulais dire que si on inverse la condition on pourrait avoir un truc genre : 
```python
if condition_inversée:
  return external_data

...le code qui était dans le if avant ...
```

Reply:
Ok c'est modifié :+1: